semantic/src/test/kotlin/tools/samt/semantic/SemanticModelTest.kt

```diff
@@ -345,7 +345,9 @@ class SemanticModelTest {
             """.trimIndent()
             parseAndCheck(
                 source to listOf(
-                    "Error: Invalid regex pattern"
+                    "Error: Invalid regex pattern: 'Dangling meta character '+' near index 5\n" +
+                            "fo/+++!hi\n" +
+                            "     ^'"
```

Comment:
Using \n in this way breaks tests on Windows 🙈 

Either replace newline characters in the test (we've done that elsewhere before) or find another creative way to solve this issue

Reply:
I have replaced the newlines with calls to `System.lineSeparator`. I don't have access to a windows machine so this needs to be verified by someone

Reply:
It is verified by the CI Pipeline 😉